### PR TITLE
Explain steps to add a new page type

### PR DESCRIPTION
**Pull request by pjlee11**
No issue, just read me info for adding a new page type. 

Could be a Wiki rather than directly in the read me but we currently don't have much of a workflow with github Wiki's ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,36 @@ If error is set to true the Error component is returned, giving the user a visua
...
+  - If required for the new page type this is where any pre-processing rules should be added. These are needed for use cases where we want to manipulate the data before it is received by the container for the page.
+    - EG: On the articles routes [unique ID's](https://github.com/bbc/simorgh/blob/2db3185cd8c5c076bc004b03bb6e8dad62b0c109/src/app/routes/getInitialData/article/index.js#L19) are added to each block in the payload
+#### 5) Add a new route to the react router config
+  - This should be done for AMP and Canoical pages together
+  - [Route example](https://github.com/bbc/simorgh/blob/2db3185cd8c5c076bc004b03bb6e8dad62b0c109/src/app/routes/index.js#L22-L28)
+#### 6) Add Cypress E2E tests for the new page type
+  - This requires config in `cypress/support/config/services.js` for every service (even if to set the new page type to undefined)
+  - If required bespoke tests for the page type should be added inside of `cypress/integration/pages/`
+ 
+NB: With this many steps it is suggested to have multiple PRs when adding a new page type as to not have a singular huge PR.
```
Comment (dr3) [inline README.md]: Can you add something about, while this is true, routing integation at e2es should be done togeather/immediately after eachother

Comment (pjlee11) [reply README.md]: I'm thinking of suggesting ...

PR 1 = fixture data + serving it
PR 2 = container + getInitialData
PR 3 = page routing + cypress tests*

* If Cypress tests are not added in the same PR as the page routing they should immediately follow the page routing PR. 

Thoughts?

Comment (dr3) [reply README.md]: PR 2 can be split if needed, as long as were clear P3 has to be togeather :)

**Review by dr3 — APPROVED** [review]
LGTM, good words

**Review by rhenshaw56 — APPROVED** [review]
Read's good!

**Pull request merged**